.github/workflows/merge.yml

```diff
@@ -44,12 +44,13 @@ jobs:
       - name: Version Beta
         if: github.ref == 'refs/heads/develop'
         run: |
-          npm run build
+          node ./scripts/update-package-json-workspaces.js
```

Comment:
changesets is looking at workspaces which points to the actual source code.

nx builds the completd code into dist/[folder]/[name] so we should modify the workspaces to point to dist/

then we remove dist/ before committing back.

Reply:
in hindsight, an option instead of this manual work of reorganizing the workspaces directories is to point the files[] in the packages/[package]/package.json  array back to dist and changesets should still publish correctly. 

example:

```
// javascript-sdk package.json
{
   ...stuff,
   "files": ["./README", "./LICENSE", "../../dist/packages/javascript-sdk/*", "./package.json"]
   ....stuff
  }

Because that way the workspace stays the same, but the "source files is still "dist". 

As long as we never "generate" the package.json and we manually upkeep the "exports" and stuff this shouldn't be an issue.


@cerebrl do you think its a better idea?

Reply:
I don't think I have enough experience with this issue to have an opinion. I'll go along with what you feel is best.